ydb/core/tx/schemeshard/schemeshard_impl.h

```diff
@@ -559,7 +559,7 @@ class TSchemeShard
     NKikimrSchemeOp::TPathVersion  GetPathVersion(const TPath& pathEl) const;
     ui64 GetAliveChildren(TPathElement::TPtr pathEl, const std::optional<TPathElement::EPathType>& type = std::nullopt) const;
 
-    const TTableInfo* GetMainTableForIndex(TPathId indexTableId) const;
+    const TTableInfo* GetMainTableForIndex(TPathId indexTableId, const TTableIndexInfo** index = nullptr) const;
```

Comment:
`std::optional<TTableIndexInfo*>`?

Reply:
Changed the code, now the comment is irrelevant.

Do not change the signature of the GetMainTableForIndex function.